include/vpnes/core/device.hpp

```diff
@@ -643,7 +643,7 @@ class CEventManager {
 		assert(eventMap.find(name) == eventMap.end());
 		typename T::CEvent *event = new typename T::template CLocalEvent<T>(
 		    name, time, enabled, device, std::forward<TArgs>(args)...);
-		events.emplace(device, event);
+		events.emplace(device, std::unique_ptr<typename T::CEvent>(event));
```

Comment:
Same as above, `T::CEvent *` can be converted to `CDeviceEvent *`, so it should be constructible